### PR TITLE
fix: remove global html font-size, adjust styles and integration tests

### PR DESCRIPTION
**Pull request by fpauer**
## Overview
Major css fix, removes font-size for the HTML element as this impose the base unit for the css. We should leave this definition to the consumers of this package. 

## Tests
[unit tests](https://github.com/awslabs/synchro-charts/runs/5255154677?check_suite_focus=true)
[integ tests](https://github.com/awslabs/synchro-charts/runs/5255154737?check_suite_focus=true)

## Legal
This project is available under the [Apache 2.0 License](http://www.apache.org/licenses/LICENSE-2.0.html).

### CODE REVIEW

**Review by diehbria — CHANGES_REQUESTED** [review]
1. Need to maintain the style, just not applied globally to the `html` tag
2. Need to update change long in the standard style explaining exactly what the breaking change was

File: packages/synchro-charts/src/globals/awsui.css [hunk 1]
```diff
@@ -16057,5 +16046,5 @@ body.awsui-modal-open {
 @media screen and (max-width: 992px) {
   .awsui .awsui-wizard__column-form .awsui-wizard__step-number {
     display: block;
-  }
```
Comment (gareth-amazon) [inline packages/synchro-charts/src/globals/awsui.css:16060]: This unclosed block is causing a build failure. This closing brace needs to be added back in

File: packages/synchro-charts/src/testing/app/routes.ts [hunk 2]
```diff
@@ -9,10 +9,6 @@ export const routes = [
     url: '/',
     component: 'testing-ground',
   },
-  {
```
Comment (gareth-amazon) [inline packages/synchro-charts/src/testing/app/routes.ts:12]: Why was this removed?

Comment (fpauer) [reply packages/synchro-charts/src/testing/app/routes.ts:12]: Because there is no tests int  the root path `/`

File: packages/synchro-charts/src/globals/awsui.css (hunk 1 above)
Comment (fpauer) [reply packages/synchro-charts/src/globals/awsui.css:16060]: Yes, I fixed that on my local, there other changes that I did not pushed yet

Comment (gareth-amazon) [reply packages/synchro-charts/src/globals/awsui.css:16060]: can you push them?

**Review by NorbertNader — CHANGES_REQUESTED** [review]
We should probably coordinate the release of this and https://github.com/awslabs/synchro-charts/pull/126

**Comment by fpauer** [comment]
> #126

let me know in which order we should merge, just need to merge CHANGELOGs.

**Comment by NorbertNader** [comment]
> > #126
> 
> let me know in which order we should merge, just need to merge CHANGELOGs.

I think we should:
1. Skip the version bump in both of these PRs
2. Merge
3. Create a new PR that bumps the major version to 2.0.0 and release them together.

**Review by gareth-amazon — APPROVED** [review]

**Review by NorbertNader — APPROVED** [review]
